packages/client/src/brands/__tests__/__snapshots__/getBrand.test.ts.snap

```diff
@@ -3,7 +3,8 @@
 exports[`brands client getBrand() should receive a client request error 1`] = `
 Object {
   "code": -1,
-  "message": "stub error",
+  "message": undefined,
```

Comment:
Is this correct? Is it missing a snapshot update?

Reply:
Related with Nelson's discussion. Fixed now :)